fix build error

Executing tasks: [:app:assembleDebug] in project C:\Users\resch\OneDrive\Projekte\Programmieren\Kotlin\Rezeptliste2

Calculating task graph as no cached configuration is available for tasks: :app:assembleDebug

FAILURE: Build failed with an exception.

* What went wrong:
Configuration cache state could not be cached: field `jvmTarget` of `org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompilerOptionsDefault` bean found in field `compilerOptions` of `org.jetbrains.kotlin.gradle.internal.KotlinJvmCompilerArgumentsContributor` bean found in field `value` of `kotlin.InitializedLazyImpl` bean found in field `compilerArgumentsContributor$delegate` of task `:app:kaptGenerateStubsDebugKotlin` of type `org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask`: error writing value of type 'org.gradle.api.internal.provider.DefaultProperty'
> Unknown Kotlin JVM target: 21

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

* Exception is:
org.gradle.internal.cc.impl.ConfigurationCacheError: Configuration cache state could not be cached: field `jvmTarget` of `org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompilerOptionsDefault` bean found in field `compilerOptions` of `org.jetbrains.kotlin.gradle.internal.KotlinJvmCompilerArgumentsContributor` bean found in field `value` of `kotlin.InitializedLazyImpl` bean found in field `compilerArgumentsContributor$delegate` of task `:app:kaptGenerateStubsDebugKotlin` of type `org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask`: error writing value of type 'org.gradle.api.internal.provider.DefaultProperty'
	at org.gradle.internal.cc.impl.problems.AbstractProblemsListener.onError(AbstractProblemsListener.kt:36)
	at org.gradle.internal.serialize.graph.AbstractIsolateContext.onError(Contexts.kt:258)
	at org.gradle.internal.serialize.graph.BeanPropertyExtensionsKt.writeNextProperty(BeanPropertyExtensions.kt:36)
	at org.gradle.internal.serialize.beans.services.BeanPropertyWriter.writeStateOf(BeanPropertyWriter.kt:54)
	at org.gradle.internal.serialize.graph.codecs.BeanCodec.writeBeanOf(BeanCodec.kt:57)
	at org.gradle.internal.serialize.graph.codecs.BeanCodec.encode(BeanCodec.kt:35)
	at org.gradle.internal.serialize.graph.CombinatorsKt$reentrant$1$encodeLoop$1.invokeSuspend(Combinators.kt:97)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlin.coroutines.ContinuationKt.startCoroutine(Continuation.kt:115)
	at org.gradle.internal.serialize.graph.CombinatorsKt$reentrant$1.encodeLoop(Combinators.kt:99)
	at org.gradle.internal.serialize.graph.CombinatorsKt$reentrant$1.encode(Combinators.kt:66)
	at org.gradle.internal.serialize.graph.codecs.BindingsBackedCodec.encode(BindingsBackedCodec.kt:62)
	at org.gradle.internal.serialize.graph.DefaultWriteContext.write(Contexts.kt:84)
	at org.gradle.internal.serialize.graph.BeanPropertyExtensionsKt.writeNextProperty(BeanPropertyExtensions.kt:34)
	at org.gradle.internal.serialize.beans.services.BeanPropertyWriter.writeStateOf(BeanPropertyWriter.kt:54)
	at org.gradle.internal.serialize.graph.CodecKt.encodeBean(Codec.kt:290)
	at org.gradle.internal.serialize.codecs.core.jos.JavaObjectSerializationCodec$WriteReplaceEncoding.encode(JavaObjectSerializationCodec.kt:169)
	at org.gradle.internal.serialize.graph.codecs.BindingsBackedCodec.encode(BindingsBackedCodec.kt:62)
	at org.gradle.internal.serialize.graph.DefaultWriteContext.write(Contexts.kt:84)
	at org.gradle.internal.serialize.graph.BeanPropertyExtensionsKt.writeNextProperty(BeanPropertyExtensions.kt:34)
	at org.gradle.internal.serialize.beans.services.BeanPropertyWriter.writeStateOf(BeanPropertyWriter.kt:54)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodec$writeTask$3$2$1.invokeSuspend(TaskNodeCodec.kt:102)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodec$writeTask$3$2$1.invoke(TaskNodeCodec.kt)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodec$writeTask$3$2$1.invoke(TaskNodeCodec.kt)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodecKt.withTaskOf(TaskNodeCodec.kt:235)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodecKt.access$withTaskOf(TaskNodeCodec.kt:1)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodec.writeTask(TaskNodeCodec.kt:96)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodec.encode(TaskNodeCodec.kt:73)
	at org.gradle.internal.serialize.codecs.core.TaskNodeCodec.encode(TaskNodeCodec.kt:66)
	at org.gradle.internal.serialize.graph.codecs.BindingsBackedCodec.encode(BindingsBackedCodec.kt:62)
	at org.gradle.internal.serialize.graph.DefaultWriteContext.write(Contexts.kt:84)
	at org.gradle.internal.serialize.codecs.core.WorkNodeCodec.doWrite(WorkNodeCodec.kt:76)
	at org.gradle.internal.serialize.codecs.core.WorkNodeCodec.writeWork(WorkNodeCodec.kt:55)
	at org.gradle.internal.cc.impl.ConfigurationCacheState.writeWorkGraphOf(ConfigurationCacheState.kt:496)
	at org.gradle.internal.cc.impl.ConfigurationCacheState.writeBuildContent$configuration_cache(ConfigurationCacheState.kt:457)
	at org.gradle.internal.cc.impl.ConfigurationCacheState.writeBuildState(ConfigurationCacheState.kt:313)
	at org.gradle.internal.cc.impl.ConfigurationCacheState.writeBuildsInTree(ConfigurationCacheState.kt:287)
	at org.gradle.internal.cc.impl.ConfigurationCacheState.writeRootBuild(ConfigurationCacheState.kt:262)
	at org.gradle.internal.cc.impl.ConfigurationCacheState.writeRootBuildState(ConfigurationCacheState.kt:160)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO$writeRootBuildStateTo$1.invokeSuspend(ConfigurationCacheIO.kt:164)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO$writeRootBuildStateTo$1.invoke(ConfigurationCacheIO.kt)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO$writeRootBuildStateTo$1.invoke(ConfigurationCacheIO.kt)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO$writeConfigurationCacheState$3$1.invokeSuspend(ConfigurationCacheIO.kt:221)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO$writeConfigurationCacheState$3$1.invoke(ConfigurationCacheIO.kt)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO$writeConfigurationCacheState$3$1.invoke(ConfigurationCacheIO.kt)
	at org.gradle.internal.serialize.graph.RunningKt$runWriteOperation$1.invokeSuspend(Running.kt:43)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlin.coroutines.ContinuationKt.startCoroutine(Continuation.kt:115)
	at org.gradle.internal.serialize.graph.RunningKt.runToCompletion(Running.kt:54)
	at org.gradle.internal.serialize.graph.RunningKt.runWriteOperation(Running.kt:42)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO.writeConfigurationCacheState(ConfigurationCacheIO.kt:220)
	at org.gradle.internal.cc.impl.ConfigurationCacheIO.writeRootBuildStateTo$configuration_cache(ConfigurationCacheIO.kt:162)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$writeConfigurationCacheState$1.run(DefaultConfigurationCache.kt:483)
	at org.gradle.internal.Factories$1.create(Factories.java:31)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withReplacedLocks(DefaultWorkerLeaseService.java:363)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$DefaultBuildProjectRegistry.withMutableStateOfAllProjects(DefaultProjectStateRegistry.java:240)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$DefaultBuildProjectRegistry.withMutableStateOfAllProjects(DefaultProjectStateRegistry.java:233)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.writeConfigurationCacheState(DefaultConfigurationCache.kt:482)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.access$writeConfigurationCacheState(DefaultConfigurationCache.kt:66)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$saveWorkGraph$1.invoke(DefaultConfigurationCache.kt:408)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$saveWorkGraph$1.invoke(DefaultConfigurationCache.kt:406)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$saveToCache$1$1.invoke(DefaultConfigurationCache.kt:424)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$saveToCache$1$1.invoke(DefaultConfigurationCache.kt:421)
	at org.gradle.internal.cc.impl.ConfigurationCacheRepository$StoreImpl$useForStore$1.invoke(ConfigurationCacheRepository.kt:189)
	at org.gradle.internal.cc.impl.ConfigurationCacheRepository$StoreImpl$useForStore$1.invoke(ConfigurationCacheRepository.kt:181)
	at org.gradle.internal.cc.impl.ConfigurationCacheRepository$withExclusiveAccessToCache$1.get(ConfigurationCacheRepository.kt:251)
	at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock(LockOnDemandCrossProcessCacheAccess.java:90)
	at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock(DefaultCacheCoordinator.java:213)
	at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock(DefaultPersistentDirectoryStore.java:149)
	at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock(DefaultCacheFactory.java:206)
	at org.gradle.internal.cc.impl.ConfigurationCacheRepository.withExclusiveAccessToCache(ConfigurationCacheRepository.kt:249)
	at org.gradle.internal.cc.impl.ConfigurationCacheRepository.access$withExclusiveAccessToCache(ConfigurationCacheRepository.kt:48)
	at org.gradle.internal.cc.impl.ConfigurationCacheRepository$StoreImpl.useForStore(ConfigurationCacheRepository.kt:181)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$saveToCache$1.invoke(DefaultConfigurationCache.kt:421)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$saveToCache$1.invoke(DefaultConfigurationCache.kt:420)
	at org.gradle.configurationcache.ConfigurationCacheBuildOperationsKt$withStoreOperation$1.run(ConfigurationCacheBuildOperations.kt:54)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
	at org.gradle.configurationcache.ConfigurationCacheBuildOperationsKt.withStoreOperation(ConfigurationCacheBuildOperations.kt:47)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.saveToCache(DefaultConfigurationCache.kt:420)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.saveWorkGraph(DefaultConfigurationCache.kt:406)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.access$saveWorkGraph(DefaultConfigurationCache.kt:66)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$loadOrScheduleRequestedTasks$1.invoke(DefaultConfigurationCache.kt:177)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache$loadOrScheduleRequestedTasks$1.invoke(DefaultConfigurationCache.kt:175)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.runWorkThatContributesToCacheEntry(DefaultConfigurationCache.kt:378)
	at org.gradle.internal.cc.impl.DefaultConfigurationCache.loadOrScheduleRequestedTasks(DefaultConfigurationCache.kt:175)
	at org.gradle.internal.cc.impl.ConfigurationCacheAwareBuildTreeWorkController$scheduleAndRunRequestedTasks$executionResult$1.apply(ConfigurationCacheAwareBuildTreeWorkController.kt:45)
	at org.gradle.internal.cc.impl.ConfigurationCacheAwareBuildTreeWorkController$scheduleAndRunRequestedTasks$executionResult$1.apply(ConfigurationCacheAwareBuildTreeWorkController.kt:44)
	at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.withNewWorkGraph(DefaultIncludedBuildTaskGraph.java:112)
	at org.gradle.internal.cc.impl.ConfigurationCacheAwareBuildTreeWorkController.scheduleAndRunRequestedTasks(ConfigurationCacheAwareBuildTreeWorkController.kt:44)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$scheduleAndRunTasks$1(DefaultBuildTreeLifecycleController.java:77)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$runBuild$4(DefaultBuildTreeLifecycleController.java:120)
	at org.gradle.internal.model.StateTransitionController.lambda$transition$6(StateTransitionController.java:169)
	at org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:266)
	at org.gradle.internal.model.StateTransitionController.lambda$transition$7(StateTransitionController.java:169)
	at org.gradle.internal.work.DefaultSynchronizer.withLock(DefaultSynchronizer.java:44)
	at org.gradle.internal.model.StateTransitionController.transition(StateTransitionController.java:169)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.runBuild(DefaultBuildTreeLifecycleController.java:117)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.scheduleAndRunTasks(DefaultBuildTreeLifecycleController.java:77)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.scheduleAndRunTasks(DefaultBuildTreeLifecycleController.java:72)
	at org.gradle.tooling.internal.provider.runner.BuildModelActionRunner.run(BuildModelActionRunner.java:53)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.internal.buildtree.ProblemReportingBuildActionRunner.run(ProblemReportingBuildActionRunner.java:49)
	at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:65)
	at org.gradle.tooling.internal.provider.FileSystemWatchingBuildActionRunner.run(FileSystemWatchingBuildActionRunner.java:140)
	at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:41)
	at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.lambda$execute$0(RootBuildLifecycleBuildActionExecutor.java:40)
	at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:130)
	at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.execute(RootBuildLifecycleBuildActionExecutor.java:40)
	at org.gradle.internal.buildtree.InitDeprecationLoggingActionExecutor.execute(InitDeprecationLoggingActionExecutor.java:62)
	at org.gradle.internal.buildtree.InitProblems.execute(InitProblems.java:36)
	at org.gradle.internal.buildtree.DefaultBuildTreeContext.execute(DefaultBuildTreeContext.java:40)
	at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.lambda$execute$0(BuildTreeLifecycleBuildActionExecutor.java:71)
	at org.gradle.internal.buildtree.BuildTreeState.run(BuildTreeState.java:60)
	at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.execute(BuildTreeLifecycleBuildActionExecutor.java:71)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:61)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:57)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor.execute(RunAsBuildOperationBuildActionExecutor.java:57)
	at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.lambda$execute$0(RunAsWorkerThreadBuildActionExecutor.java:36)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:267)
	at org.gradle.internal.work.DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:131)
	at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.execute(RunAsWorkerThreadBuildActionExecutor.java:36)
	at org.gradle.tooling.internal.provider.continuous.ContinuousBuildActionExecutor.execute(ContinuousBuildActionExecutor.java:110)
	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecutor.execute(SubscribableBuildActionExecutor.java:64)
	at org.gradle.internal.session.DefaultBuildSessionContext.execute(DefaultBuildSessionContext.java:46)
	at org.gradle.internal.buildprocess.execution.BuildSessionLifecycleBuildActionExecutor$ActionImpl.apply(BuildSessionLifecycleBuildActionExecutor.java:92)
	at org.gradle.internal.buildprocess.execution.BuildSessionLifecycleBuildActionExecutor$ActionImpl.apply(BuildSessionLifecycleBuildActionExecutor.java:80)
	at org.gradle.internal.session.BuildSessionState.run(BuildSessionState.java:71)
	at org.gradle.internal.buildprocess.execution.BuildSessionLifecycleBuildActionExecutor.execute(BuildSessionLifecycleBuildActionExecutor.java:62)
	at org.gradle.internal.buildprocess.execution.BuildSessionLifecycleBuildActionExecutor.execute(BuildSessionLifecycleBuildActionExecutor.java:41)
	at org.gradle.internal.buildprocess.execution.StartParamsValidatingActionExecutor.execute(StartParamsValidatingActionExecutor.java:64)
	at org.gradle.internal.buildprocess.execution.StartParamsValidatingActionExecutor.execute(StartParamsValidatingActionExecutor.java:32)
	at org.gradle.internal.buildprocess.execution.SessionFailureReportingActionExecutor.execute(SessionFailureReportingActionExecutor.java:51)
	at org.gradle.internal.buildprocess.execution.SessionFailureReportingActionExecutor.execute(SessionFailureReportingActionExecutor.java:39)
	at org.gradle.internal.buildprocess.execution.SetupLoggingActionExecutor.execute(SetupLoggingActionExecutor.java:47)
	at org.gradle.internal.buildprocess.execution.SetupLoggingActionExecutor.execute(SetupLoggingActionExecutor.java:31)
	at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:70)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:29)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput.lambda$execute$0(ForwardClientInput.java:40)
	at org.gradle.internal.daemon.clientinput.ClientInputForwarder.forwardInput(ClientInputForwarder.java:80)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:64)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:84)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)
	at org.gradle.launcher.daemon.server.DaemonStateCoordinator.lambda$runCommand$0(DaemonStateCoordinator.java:320)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:48)
Caused by: java.lang.IllegalArgumentException: Unknown Kotlin JVM target: 21
	at org.jetbrains.kotlin.gradle.dsl.JvmTarget$Companion.fromTarget(JvmTarget.kt:26)
	at org.jetbrains.kotlin.gradle.tasks.DefaultKotlinJavaToolchain$wireJvmTargetToToolchain$1$1.invoke(DefaultKotlinJavaToolchain.kt:78)
	at org.jetbrains.kotlin.gradle.tasks.DefaultKotlinJavaToolchain$wireJvmTargetToToolchain$1$1.invoke(DefaultKotlinJavaToolchain.kt:70)
	at org.jetbrains.kotlin.gradle.tasks.DefaultKotlinJavaToolchain$sam$org_gradle_api_Transformer$0.transform(DefaultKotlinJavaToolchain.kt)
	at org.gradle.api.internal.provider.ValueSupplier$Present.transform(ValueSupplier.java:518)
	at org.gradle.api.internal.provider.TransformBackedProvider.mapValue(TransformBackedProvider.java:91)
	at org.gradle.api.internal.provider.TransformBackedProvider.calculateExecutionTimeValue(TransformBackedProvider.java:73)
	at org.gradle.api.internal.provider.MappingProvider.calculateExecutionTimeValue(MappingProvider.java:57)
	at org.gradle.internal.serialize.codecs.core.FixedValueReplacingProviderCodec.encodeProvider(ProviderCodecs.kt:93)
	at org.gradle.internal.serialize.codecs.core.PropertyCodec.encode(ProviderCodecs.kt:333)
	at org.gradle.internal.serialize.codecs.core.PropertyCodec.encode(ProviderCodecs.kt:325)
	at org.gradle.internal.serialize.graph.codecs.BindingsBackedCodec.encode(BindingsBackedCodec.kt:62)
	at org.gradle.internal.serialize.graph.DefaultWriteContext.write(Contexts.kt:84)
	at org.gradle.internal.serialize.graph.BeanPropertyExtensionsKt.writeNextProperty(BeanPropertyExtensions.kt:34)
	... 170 more


Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.9/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1s
Configuration cache entry discarded due to serialization error.
---

./gradlew build --warning-mode all
Calculating task graph as no cached configuration is available for tasks: build

> Configure project :app
The org.gradle.api.plugins.Convention type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/8.9/userguide/upgrading_version_8.html#deprecated_access_to_conventions
The Configuration.fileCollection(Spec) method has been deprecated. This is scheduled to be removed in Gradle 9.0. Use Configuration.getIncoming().artifactView(Action) with a componentFilter instead. Consult the upgrading guide for further information: https://docs.gradle.org/8.9/userguide/upgrading_version_8.html#deprecate_filtered_configuration_file_and_filecollection_methods

FAILURE: Build failed with an exception.

* What went wrong:
Configuration cache state could not be cached: field `jvmTarget` of `org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompilerOptionsDefault` bean found in field `compilerOptions` of `org.jetbrains.kotlin.gradle.internal.KotlinJvmCompilerArgumentsContributor` bean found in field `value` of `kotlin.InitializedLazyImpl` bean found in field `compilerArgumentsContributor$delegate` of task `:app:kaptGenerateStubsDebugKotlin` of type `org.jetbrains.kotlin.gradle.internal.KaptGenerateStubsTask`: error writing value of type 'org.gradle.api.internal.provider.DefaultProperty'
> Unknown Kotlin JVM target: 23

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 4s
Configuration cache entry discarded due to serialization error.

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -11,7 +11,7 @@ android {
     defaultConfig {
         applicationId "com.example.rezeptliste2"
         minSdk 21
-        targetSdk 33
+        targetSdk 34
         versionCode 1
         versionName "1.0"
         javaCompileOptions {
@@ -49,7 +49,6 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
-    buildToolsVersion '35.0.0'
 }
 
 dependencies {

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -49,6 +49,7 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
+    buildToolsVersion '35.0.0'
 }
 
 dependencies {

--- a/gradle.properties
+++ b/gradle.properties
@@ -23,4 +23,4 @@ kotlin.code.style=official
 android.nonTransitiveRClass=true
 android.nonFinalResIds=false
 org.gradle.configuration-cache=true
-kotlin.jvm.target=17
+kotlin.compiler.jvmTarget=17

